recipes/libpcsclite/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 856a1daed9a5ae1ac265c75ce9f81743e66a972e5be5a1d3a2196ed23f75ea26
+
+build:
```

There should also be a run_export section which pins this library according to it's ABI compatability.

---

Do you have more guidance about this part? I'm unfamiliar

---

According to the ABI lab (which is outdated) https://abi-laboratory.pro/index.php?view=timeline&l=pcsc-lite it's mostly backward compatible, so I guess this is enough:
```suggestion
build:
  run_exports:
    - {{ pin_subpackage('libpcsclite') }}
```
This ensures any packages built against `libpcsclite` v1.x will use v1.x.